content/posts/newsletter-019/index.md

```diff
@@ -383,12 +383,12 @@ screen.
 _Macroquad-UI runtime GUI skins configuration._
 
 [macroquad] is a cross-platform (Windows/Linux/macOS/Android/iOS/WASM)
-game framework built on top of [miniquad].  
+game framework built on top of [miniquad].
```

Comment:
Not sure why this is showing up in the diff, but for some reason I can't fix it. My editor removes the whitespacing

Reply:
Your editor is probably configured to remove all pending whitespaces. Reverted these changes